### PR TITLE
Fixes adjust_cardinality to handle agg_has_view arrays. Fixes marc da…

### LINKED ISSUE #128: Fix cho_date and agg_has_view "must be string" errors in harvard and penn data
Try to run the current configs on 'penn' or 'harvard' for specific examples

### PR DESCRIPTION
**Pull request by justinlittman**
…tes to make strings.

closes #128

### CODE REVIEW

File: lib/adjust_cardinality.rb [hunk 1]
```diff
@@ -32,16 +32,23 @@ def flatten_top_level(attributes)
...
   def flatten_web_resources(attributes)
-    flatten = %w[agg_is_shown_at agg_is_shown_by agg_preview]
+    flatten = %w[agg_is_shown_at agg_is_shown_by agg_preview agg_has_view]
     attributes.except(*flatten).tap do |output|
       flatten.each do |field|
         next unless attributes.key?(field)
 
-        output[field] = process_web_resource(attributes.fetch(field))
+        # If agg_has_view, this is an array
+        output[field] = if attributes.fetch(field).is_a?(Array)
```
Comment (jcoyne) [inline lib/adjust_cardinality.rb]: can you extract this to a method?

File: lib/adjust_cardinality.rb [hunk 2]
```diff
@@ -32,16 +32,23 @@ def flatten_top_level(attributes)
     end
   end
 
+  # rubocop:disable Metrics/MethodLength
   def flatten_web_resources(attributes)
-    flatten = %w[agg_is_shown_at agg_is_shown_by agg_preview]
+    flatten = %w[agg_is_shown_at agg_is_shown_by agg_preview agg_has_view]
     attributes.except(*flatten).tap do |output|
       flatten.each do |field|
         next unless attributes.key?(field)
```
Comment (jcoyne) [inline lib/adjust_cardinality.rb]: If we do
```ruby
value = attributes[field]
next unless value
```
then you don't need to call `attributes.fetch(field)` below.

File: lib/adjust_cardinality.rb (hunk 1 above)
Comment (justinlittman) [reply lib/adjust_cardinality.rb]: I thought about that but couldn't come up with a name for a method that was less confusing than just if/then code. Suggestions?

Comment (jcoyne) [reply lib/adjust_cardinality.rb]: I'd call it `flatten_web_resource`  as this is for one item in the list.

**Pull request merged**